Source/WebKit/UIProcess/Cocoa/WebPageWebView.swift

```diff
@@ -46,13 +46,23 @@ public final class WebPageWebView: WKWebView {
         delegate?.findInteraction(interaction, didEnd: session)
     }
 
+    #if USE_APPLE_INTERNAL_SDK
     override func supportsTextReplacement() -> Bool {
         guard let delegate else {
             return super.supportsTextReplacement()
         }
 
         return super.supportsTextReplacement() && delegate.supportsTextReplacement()
     }
+    #else
+    override var supportsTextReplacement: Bool {
+        guard let delegate else {
+            return super.supportsTextReplacement
+        }
+
+        return super.supportsTextReplacement && delegate.supportsTextReplacement()
+    }
+    #endif // USE_APPLE_INTERNAL_SDK
```

Comment:
Seems like we shouldn't need to do this; it should be the same definition in both internal and public builds. Even though it probably works both ways since the ObjC representation of a zero-argument member function and a property getter are the same.

Perhaps there's a problem with a UIKit SPI header that we need to adjust?

Reply:
Yeah, that's^ the problem. This is rdar://95125552 (cc @pxlcoder).